packages/drivers/odsp-driver/src/test/odspDriverUrlResolver2Tests.spec.ts

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable max-len */
+/* eslint-disable dot-notation */
+
+import { strict as assert } from "assert";
+import sinon from "sinon";
+import { IRequest } from "@fluidframework/core-interfaces";
+import { OdspDriverUrlResolver2 } from "../odspDriverUrlResolver2";
+import { SharingLinkScopeFor, SharingLinkTokenFetcher } from "../tokenFetch";
+import { getHashedDocumentId } from "../odspUtils";
+import { createOdspUrl } from "../createOdspUrl";
+import * as graph from "../graph";
+import { getLocatorFromOdspUrl, storeLocatorInOdspUrl } from "../odspFluidFileLink";
+import { IOdspResolvedUrl, SharingLinkHeader } from "../contracts";
+
+describe("OdspUrlResolver3", () => {
```

Comment:
Can you please put better name here? I do not know what 3 stands here for

Reply:
Renamed